pages_builder/pages/big-stat.yml

```diff
@@ -0,0 +1,9 @@
+pageTitle: Big statistic
+pageDescription: Used to prominently display statistics
+assetPath: govuk_template/assets/
+grid: column-one-whole
+examples:
+  - |
+    {% import "big-stat.html" as big_stat %}
```

Comment:
`big-stat.html` should already be imported for you. It'll get called once for each example you specify. See other files in the same dir for examples. It describes it a little bit in the README in the `Templates` section on line 72 (can't seem to link to it)